transform-plugins/src/main/java/io/cdap/plugin/CSVFormatter.java

```diff
@@ -116,6 +119,17 @@ public void configurePipeline(PipelineConfigurer pipelineConfigurer) throws Ille
   @Override
   public void prepareRun(StageSubmitterContext context) throws Exception {
     super.prepareRun(context);
+
+    Schema schema = context.getInputSchema();
```

Comment:
This is mostly wrong. From the transform method, https://github.com/cdapio/hydrator-plugins/blob/develop/transform-plugins/src/main/java/io/cdap/plugin/CSVFormatter.java#L181, this basically converts all the input fields to the first field of the output schema, your field lineage should reflect this.